lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -21,7 +21,9 @@ class Modules
             '-I' => [false, 'Ignore the command if the only match has the same name as the search'],
             '-o' => [true,  'Send output to a file in csv format'],
             '-S' => [true,  'Regex pattern used to filter search results'],
-            '-u' => [false, 'Use module if there is one result']
+            '-u' => [false, 'Use module if there is one result'],
+            '-s' => [true, 'Sort search results'],
+            '-r' => [true, 'Work with sort to reverse the order of search resulsts']
```

Comment:
```suggestion
            '-r' => [true, 'Reverse the order of search results']
```

Reply:
For the `-s` option it might be good to denote if it defaults to ascending or descending order by default. Looks like by default you are sorting in ascending order.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/14917/commits/4382aeed28bccdfec455782e95fda9674f8490a4 with descriptions on all outputs of the `-s` and `-r` options to explain what ordering effect they have.